src/components/hero/_macro-options.md

```diff
@@ -1,17 +1,17 @@
-| title                   | string                                                    | true     | Text for the hero title                                                                                                 |
-| subtitle                | string                                                    | false    | Text for the hero subtitle                                                                                              |
-| text                    | string                                                    | false    | Text to follow the hero title and subtitle                                                                              |
-| button                  | `Object<Button>`                                          | false    | Settings for the hero [call to action button](#button)                                                                  |
-| html                    | string                                                    | false    | Allows arbitrary HTML for additional content to be added to the component                                               |
-| detailsColumns          | integer                                                   | false    | Number of grid columns for the hero to span on screens larger than the medium breakpoint, defaults to 8                 |
-| descriptionList         | `DescriptionList` [_(ref)_](/components/description-list) | false    | Settings to set the DescriptionList component within the HTML `<hero>` element                                          |
-| officialStatisticsBadge | boolean                                                   | false    | Set to “true” display the official statistics badge in the hero                                                         |
-| topic                   | string                                                    | false    | Topic for the hero                                                                                                      |
-| breadcrumbs             | `Breadcrumbs` [_(ref)_](/components/breadcrumbs)          | false    | Settings to set the Breadcrumbs component within the HTML `<hero>` element                                              |
+| Name                    | Type                                                      | Required | Description                                                                                                                                |
+| ----------------------- | --------------------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| variants                | array or string                                           | false    | An array of values or single value (string) to adjust the component using available variant, “dark, navy-blue, grey and pale-blue-bubbles” |
```

Comment:
Ive just realised that to be consistent with the navy-blue variant we should rename this variant to just "pale-blue". The navy blue variant has bubbles but doesn't mention them in the name. So all references will need to be updated